proposals/0xxx-enum-cases-as-protocol-witnesses.md

```diff
@@ -0,0 +1,211 @@
+# Enum cases as protocol witnesses
+
+* Proposal: [SE-XXXX](xxxx-enum-cases-as-protocol-witnesses.md)
+* Author: [Suyash Srijan](https://www.github.com/theblixguy)
```

Comment:
The `www.` can produce a warning in <https://data.swift.org/swift-evolution/proposals>

```suggestion
* Author: [Suyash Srijan](https://github.com/theblixguy)
```